project/webapp/controller/ProductPage.controller.js

```diff
@@ -255,6 +256,11 @@ sap.ui.define(
         this.oMessagePopover.toggle(oPopoverBtn);
       },
 
+      onCommentPost(sFeedValue) {
```

Comment:
Get feed value using common approach like oEvent.getSource().getValue()
Usually we I trying to avoid the putting params directly into function in XML view